playbooks/conferences.md

```diff
@@ -24,6 +24,30 @@ guidelines:
 - If you need help preparing a conference talk, stop by [Ash][ash]'s weekly Writing Office Hours.
 - It's also recommended sharing the talk and lessons with the team.
 
+## Approval Process
+
+- Find an event (conference/seminar/course) that you would like to attend. Identify items in the agenda that will
+  contribute to both your growth goals and Artsy's business goals.
+- Estimate any travel or lodging expenses that may be needed to attend the event using Concur to search for airfare
+  and hotels.
+- Speak to your manager about attending the event before purchasing tickets. Communicate the following items:
+  - How will this event contribute to your and your team's goals?
+  - What are the dates of the event?
+  - What is the attendance fee for the event?
+  - What travel and lodging fees will be required?
+  - Managers should consider the following:
+    - Does the event align with individual and business goals?
+    - Does the attendance cost fit within the remaining budget?
+    - Is attendance being requested far enough in advance to keep costs low?
+    - Have enough engineers been given a chance to attend events?
+- Purchase tickets for the event, and start an expense report in Concur or Vistra.
+- Make travel and loding arrangements via Concur and submit the expense report for approval.
+  - Managers will need to approve travel and lodging expenses above a certain rate, which in some cases can put a
+    24-hour hold on itineraries pending approval.
+  - For more information, review the [2019 Travel & Expense Policy][2019-travel-and-expense-policy].
```

Comment:
Related to the linked travel and expense policy, which expenses are applicable for conference travel?

Are engineers allowed to expense local transportation and meals on days they're attending a conference? It would be great to clarify the policy in this document.

Reply:
I just want to caution that we defer to the company policy document as the ultimate source of truth here. If there are holes in that doc, I'd rather see that improve so that the wider Artsy team benefits from the clarifications, ya know?

Reply:
Conference attendance is part of our job at Artsy, so clarifying that conference travel fall under the T&E playbook's definition of "Business Travel" would be a good idea 👍 That way, it's clear that the playbook's sections for per diems and ground transportation apply.